data/src/main/java/ca/bc/gov/data/datasource/remote/LaboratoryRemoteDataSource.kt

```diff
@@ -119,4 +120,50 @@ class LaboratoryRemoteDataSource @Inject constructor(
 
         return response.toDto()
     }
+
+    suspend fun getLabTestInPdf(
+        token: String,
+        hdid: String,
+        reportId: String,
+        isCovid19: Boolean
+    ): LabTestPdfResponse {
+
+        val response = safeCall {
+            laboratoryApi.getLabTestInPdf(
+                token,
+                hdid,
+                reportId,
+                isCovid19
+            )
+        }
+            ?: throw MyHealthException(SERVER_ERROR, MESSAGE_INVALID_RESPONSE)
+
+        if (response.error != null) {
+            throw MyHealthException(SERVER_ERROR, MESSAGE_INVALID_RESPONSE)
+        }
+
+        if (isLabTestPdfResponseValid(response)) {
```

Comment:
Add ! in if condition like below:
if (!isLabTestPdfResponseValid(response)) {
Because when response is invalid then we need to throw error.